### ISSUE TITLE
shadow on smooth scrolling

### ISSUE THREAD

**Issue by aryanrajseo** [Caldis/Mos#652]
Hi, I am using LG 4k monitor 50hertz and i see a shadow when a smooth scrolling is enabled. is it normal?

https://github.com/Caldis/Mos/assets/34836534/29cbe552-2710-4554-8e8d-8cbb79e03e83

**System Info | 系统信息:**
 - Mouse: Logi signature 650
 - OS: MacOS Sonoma
 - Browser Chrome

**Comment by Fejm** [comment]
There is no shadow on your video. What you see is drawn on your screen as is has low reaction time. Google: screen ghosting

**Comment by Caldis** [comment]
I think you should check your HDMI/DP cable first.

**Comment by aryanrajseo** [comment]
yes, seems like the hdmi or screen slow response, moved to dpcable previously and seems like it get solved.

**Comment by landsman** [comment]
Interesting! Thanks for these responses. I had exactly the same issue (mac vs. hdmi DELL monitor via HDMI).
